@xen-orchestra/web-core/lib/components/ui/alert/UiAlert.vue

```diff
@@ -0,0 +1,93 @@
+<!-- v3 -->
+<template>
+  <div :class="toVariants({ accent })" class="ui-alert">
+    <div class="content">
+      <VtsIcon class="information-icon" :accent :icon="faCircle" :overlay-icon="icon" />
+      <div class="alert">
+        <div class="typo-body-regular-small">
+          <slot />
+        </div>
+        <div v-if="slots.description" class="typo-body-regular-small">
+          <slot name="description" />
+        </div>
+      </div>
+      <UiButtonIcon v-if="close" class="close-icon" :icon="faXmark" accent="brand" size="medium" />
```

Comment:
```suggestion
      <UiButtonIcon v-if="close" class="close-button" :icon="faXmark" accent="brand" size="medium" />
```
Also, I think the button should emit an event when clicked.

Reply:
:see_no_evil: